### DISCUSSION TITLE
3Dmol.js package install into docker container

### DISCUSSION THREAD

**Discussion by kanyi** [requarks/wiki#7283] [Help / Questions]
OS: Debian 12 and Docker
Wiki.js version: 2.5.300
Database engine: PostgreSQL 15`

Hi. It would be nice to see 3D chemical structures in wiki.js. There is JavaScript programs for this. Like 3Dmol.js. I try to using it.
The simple script method working. In Page Properties \ Scripts place these two line:

```
<script src="https://3Dmol.org/build/3Dmol-min.js"></script>
<script src="https://3Dmol.org/build/3Dmol.ui-min.js"></script>
```

And I can visualise molecules.
After this, I try to install in the docker container the 3Dmol.js program, 
```
npm install 3Dmoljs
```

but this gave me error. I attached the error message: [2024-07-29T13_49_42_453Z-debug-0.log](https://github.com/user-attachments/files/16415374/2024-07-29T13_49_42_453Z-debug-0.log)

What I am doing wrong? I'm sure there is a better solution for this.

**Comment by Eddy08** [comment]
Any update regarding this? 
Can you share your package.json if it's still an issue ? 
I hope you have tried running --force, did you get any further lead on that.
